### ISSUE TITLE
Declarations should come from SDK

### ISSUE THREAD

**Issue by danmarshall** [Microsoft/BotFramework-DirectLineJS#14]
1. The [BotBuilder SDK declarations](https://github.com/Microsoft/BotBuilder/blob/master/Node/core/src/interfaces.d.ts) should be published to DefinitelyTyped
2. This repo should get it from @types

**Comment by billba** [comment]
@types is just for packages that don't provide their own types. BotBuilder provides its own.

But I must not understand your point. This is a client library. BotBuilder is a server library. Why would one have dependencies on the other?

I wonder if maybe you are talking about Activity formats? If so, then I could imagine a type declaration file that would sit in its own project that both could depend on.

**Comment by danmarshall** [comment]
Yeah I guess it could be the Activity formats. Trying to find the overlap in the schema of stuff you write in a bot and stuff that is received on DL.
